mod-orders/mod-orders.postman_collection.json

```diff
@@ -6505,65 +6565,1068 @@
+									"name": "Put order - vendor not found",
+									"event": [
+										{
+											"listen": "prerequest",
+											"script": {
+												"id": "d8c9ac0f-96ad-4626-b6cd-5bd280e0e1a6",
+												"exec": [
+													"let utils = eval(globals.loadUtils);",
+													"",
+													"pm.variables.set(\"nonExistVendorId\", \"1a54b431-2e4f-452d-9cae-9cee66c9a892\");",
+													"",
+													"pm.sendRequest(pm.variables.get(\"mod-ordersResourcesURL\") + \"/po_listed_print_monograph.json\", (err, res) => {",
+													"    let json = utils.prepareOpenOrderWithoutPoNumber(res.json());",
+													"    json.vendor = \"{{nonExistVendorId}}\";",
+													"    json.poNumber = \"anyPoNumber\";",
+													"    json.compositePoLines[0].eresource.accessProvider = \"{{activeVendorId}}\";",
+													"    json.compositePoLines[1].eresource.accessProvider = \"{{activeVendorId}}\";",
+													"    pm.variables.set(\"po_with_non_exist_vendor\", JSON.stringify(json));",
+													"});"
+												],
+												"type": "text/javascript"
+											}
+										},
+										{
+											"listen": "test",
+											"script": {
+												"id": "4a638bb8-dbd4-4c76-a67b-a496016ee9df",
+												"exec": [
+													"pm.test(\"Status code is 422 - Unprocessable Entity\", function () {",
+													"    pm.response.to.have.status(422);",
+													"});",
+													"",
+													"pm.test(\"Expected errors verification\", function () {",
+													"    ",
+													"    pm.expect(pm.response.text()).to.include(\"Order cannot be open as the associated vendor not found\");",
+													"    pm.expect(pm.response.text()).to.include(\"vendorNotFound\");",
+													"    ",
+													"    pm.expect(pm.response.text()).to.include(pm.variables.get(\"nonExistVendorId\"));",
+													"    ",
+													"    pm.response.to.have.jsonBody(\"total_records\", 1);",
+													"});"
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"method": "PUT",
+										"header": [
+											{
+												"key": "X-Okapi-Tenant",
+												"value": "{{xokapitenant}}"
+											},
+											{
+												"key": "Content-Type",
+												"value": "application/json"
+											},
+											{
+												"key": "x-okapi-token",
+												"value": "{{xokapitoken}}"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": "{{po_with_non_exist_vendor}}"
+										},
+										"url": {
+											"raw": "{{protocol}}://{{url}}:{{okapiport}}/orders/composite-orders/{{completeOpenOrderId}}",
+											"protocol": "{{protocol}}",
+											"host": [
+												"{{url}}"
+											],
+											"port": "{{okapiport}}",
+											"path": [
+												"orders",
+												"composite-orders",
+												"{{completeOpenOrderId}}"
```

Comment:
Please use `negativeTestsOpenOrderId` instead of `completeOpenOrderId` for negative tests.